src/apps/content-editor/src/app/views/ItemEdit/Content/Actions/Widgets/Unpublish/Unpublish.js

```diff
@@ -32,22 +29,23 @@ export const Unpublish = memo(function Unpublish(props) {
   };
 
   return (
-    <CollapsibleCard
-      className={"Unpublish"}
-      header={
-        <Fragment>
-          <FontAwesomeIcon icon={faUnlink} />
-          &nbsp;Unpublish
-        </Fragment>
-      }
-    >
-      <CardContent>
-        <p>
+    <Accordion sx={{ m: "16px !important" }}>
+      <AccordionSummary
+        expandIcon={<ExpandMoreIcon />}
+        aria-controls="panel1a-content"
+        id="panel1a-header"
+      >
+        <Typography sx={{ display: "flex", alignItems: "center" }}>
+          {" "}
+          <UnpublishedIcon fontSize="small" /> Unpublish
+        </Typography>
+      </AccordionSummary>
+      <AccordionDetails>
+        <Typography>
           By unpublishing this content it will no longer be served if the URL is
           requested. The URL will return a 404 not found response.
-        </p>
-      </CardContent>
-      <CardFooter>
```

Comment:
Footer buttons seem more fitting to be wrapped within <AccordionActions>

Reply:
@agalin920 Wrapped within Typography?